project/src/main/webapp/shared.js

```diff
@@ -5,30 +5,59 @@ async function isLoggedIn() {
 }
 
 async function getPrefilledInformation(informationCategory) {
-  const response = 
-      await fetch('/prefilled-information?' + new URLSearchParams({'category': informationCategory}))
+  const response =
+      await fetch(
+          `/prefilled-information?${new URLSearchParams({'category': informationCategory})}`);
   return response.json();
 }
 
-function buildTag(tagClass) {
+function buildTagWithAdder(tagClass, text) {
```

Comment:
Could you JSDoc this function? and some of the ones that are a bit less clear in this file (i think the ones in profile,js are fairly self explanatory).

Reply:
Done